mne/preprocessing/ica.py

```diff
@@ -884,7 +884,7 @@ def _transform_evoked(self, evoked):
         if not hasattr(self, 'mixing_matrix_'):
             raise RuntimeError('No fit available. Please fit ICA.')
 
-        picks = pick_types(evoked.info, include=self.ch_names, exclude='bads',
+        picks = pick_types(evoked.info, include=self.ch_names, exclude=[],
```

Comment:
but now if I have marked bads and I don't pass picks to ICA.fit will it still ignore the bad channels? It should.

Reply:
It does. See below for the additional testing.